mite/cli/test.py

```diff
@@ -207,25 +217,54 @@ def human_readable_bytes(size):
     return size, "PB"
 
 
-def benchmark_report(http_stats_output):
-    table = PrettyTable()
-    table.field_names = ["Metric", "Avg", "Min", "Max", "Std Dev", "+/- Std Dev"]
-    number_format = (
-        lambda number: f"{number / 1000:.2f}K" if number >= 1000 else f"{number:.2f}"
-    )
+def benchmark_report(http_stats_output, has_error=False):
+    perctile_thresh_results_list = http_stats_output.percentiles_list_resp_time_store
 
-    table.add_row(
+    percentiles_headers = [
+        f"{int(percentile[0])} %tile" for percentile in perctile_thresh_results_list
+    ]
+    standard_headers = ["", "Avg", "Min", "Max", "Std Dev", "+/- Std Dev"]
+
+    latency_table = PrettyTable()
+    latency_table.field_names = standard_headers + percentiles_headers
+    latency_table.add_row(
         [
             "Latency",
             f"{http_stats_output.mean_resp_time * 1000:.2f}ms",
             f"{http_stats_output._resp_time_min * 1000:.2f}ms",
             f"{http_stats_output._resp_time_max * 1000:.2f}ms",
             f"{http_stats_output.resp_time_standard_deviation * 1000:.2f}ms",
             f"{http_stats_output.resp_time_within_standard_deviation:.2f}%",
-        ]
+            *[
+                f"\033[91m{result * 1000:.2f}ms\033[0m"
```

Comment:
This block looks a little messy, I wonder if we can find a way to make it look cleaner

Reply:
move it out into `colorise_result` function